catalog-info.yaml

```diff
@@ -436,11 +436,15 @@ spec:
       repository: elastic/logstash
       pipeline_file: ".buildkite/exhaustive_tests_pipeline.yml"
       provider_settings:
-        trigger_mode: none
-      cancel_intermediate_builds: true
-      skip_intermediate_builds: true
+        build_branches: true
```

Comment:
this means that in addition to merge commits, the pipeline will be triggered when we push a new branch, e.g. `8.14`. I don't know if this is something we want, it felt right to me.

Reply:
Agreed, as long as we exclude `docs` changes (if possible)